### PR TITLE
:circus_tent:

### PR DESCRIPTION
**Pull request by cpz**
1. Minor changes in CUtlMemoryPoolBase
2. Update CUtlTsHash, so it more proper
3. Schema was updated and tested on regression (e.g. Underlords)

### CODE REVIEW

File: include/sdk/interfaces/schemasystem/schema.h [hunk 1]
```diff
@@ -665,6 +725,10 @@ class CSchemaPtrMap {
 public:
     CUtlMap<K, V> m_Map;
     CThreadFastMutex m_Mutex;
+
+#if not defined(CS2) && not defined(DOTA2)
```
Comment (es3n1n) [inline include/sdk/interfaces/schemasystem/schema.h:729]: please... we aren't coding in python
```suggestion
#if !defined(CS2) && !defined(DOTA2)
```

**Review by es3n1n — CHANGES_REQUESTED** [review]
other than that lgtm

**Review by es3n1n — APPROVED** [review]
oops, wanted to click on this button instead

File: include/sdk/interfaces/schemasystem/schema.h (hunk 1 above)
Comment (cpz) [reply include/sdk/interfaces/schemasystem/schema.h:729]: https://www.youtube.com/watch?v=k13LlCabKdQ

**Review by es3n1n — APPROVED** [review]

**Pull request merged**